packages/web-components/src/components/va-button/va-button.tsx

```diff
@@ -138,6 +138,35 @@ export class VaButton {
       return;
     }
     this.handleClick();
+    /**
+     * If the button has a 'submit' attribute:
+     * creates a faux, invisible button, looks for the closest anceston FORM tag of button
+     * if found it appends the faux button to the DOM, then clicks and removes it.
+     * this serves to submit the form as well as running the onsubmit script for the form
+     * ALTERNATIVELY (commented out)
+     * we could the closest anceston FORM tag of button and run the form's submit method.
+     * with the caviat that this will submit the form , but not theactual onsumbit script.
+     * This is an intentional feature of HTML.
+     */
+    if (this.submit) {
+      // this.el.closest('form')?.submit();
+      const fakeButton = document.createElement('button');
+      fakeButton.innerHTML = 'auto submit';
+      fakeButton.type = 'submit';
+      fakeButton.setAttribute('style', 'display:none;');
+      if (this.el.closest('form')?.appendChild(fakeButton)) {
+        fakeButton.click();
+        fakeButton.remove();
```

Comment:
The way teams can get around `submit` not bubbling up to the `form` element is to just use the existing native `onClick` like the example in the [original issue](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2379):

```
handleSubmit = (event) => {
  event.preventDefault();
  document.myForm.submit();
}

<form>
   <va-button text="Submit form" onClick={(event) => handleSubmit(event)} />
</form>
```

So I guess my question is do we actually _need_ the native `submit`? 

How many instances do we have right now where submit is being used with va-button?

Reply:
That's the alternate solution  presented in:
`      // this.el.closest('form')?.submit();
`
The thing is, in HTML, calling a form's submit() method has a different behavior than  clicking on a submit button.  Both aproaches do submit the form successfully tho.  It's just this current one allows for  VFS team to execute a function inside the submit property of the form element as well.

I will look into the number of instances  ( which also raises the question, as long as we are at this,  should 'submit' be added to analytics-- currently it seems we get  data  on 'primary'/'secondary'.